javascript/javascript1/week4/voiceAssistance.js

```diff
@@ -0,0 +1,168 @@
+//declare variables
+let name1 = "  ";
+let toDoList = [];
+
+// number declared as global variable for simple math part. 
+let a = 10;
+let b = 12;
+
+function getReply(command){
+
+//general greeting message
+if (command.includes ("hello", "hey")) {
+    return "Hello! Welcome home."
+} 
+
+else if (command.includes("I am ")){
+    return "Hello! Nice to meet you."
+}
+
+//personal greeting message
+else if (command.includes("name"))
+{   
+    name1 = command.split(" ").pop();
+    return "Hello " + name1 + " Welcome home!! How was your day?"
+}
+
+//answer the question    
+else if (command.includes ("What is my name") || command.includes("remember")) {
+    return name1
+}
+
+//Empty string    
+else if (command === " ") {
+    return "Hi there!! May I know your name"
+}
+
+//update to-do list
+else if (command === "Add fishing to my todo") {
+    toDoList.push("fishing");
+    return toDoList[0] + " added to your todo List."
+}
+
+else if (command === "Add singing in the shower to my todo") {
+    toDoList.push("singing in the shower");
+    return toDoList[1] + " added to your todo List."
+}
+
+else if (command === "Add shopping to my todo") {
+    toDoList.push("shopping");
+    return toDoList[2] + " added to your todo List."
+}
+
```

Comment:
you can extract the activity to be pushed from the text given. Otherwise, if i want to add 'cleaning' to my todo, your code breaks. 
Also, before pushing, i would check to see if the activity is already there. So if i mistakenly add shopping twice, it only counts once.
This also ensures that if you delete shopping, it is no longer on the list :)